src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/TrainVariantAnnotationsModel.java

```diff
@@ -0,0 +1,571 @@
+                            numNegativeTrainingFromUnlabeledVariantType, variantTypeString, scoreThreshold));
+
+                    final double[][] negativeTrainingAndVariantTypeAnnotations = concatenateLabeledAndUnlabeledNegativeTrainingData(
+                            annotationNames, annotations, unlabeledAnnotations, isNegativeTrainingFromLabeledTrainingAndVariantType, isNegativeTrainingFromUnlabeledVariantType);
+                    final int numNegativeTrainingAndVariantType = negativeTrainingAndVariantTypeAnnotations.length;
+                    final List<Boolean> isNegativeTrainingAndVariantType = Collections.nCopies(numNegativeTrainingAndVariantType, true);
+
+                    logger.info(String.format("Training %s negative model with %d negative-training sites x %d annotations %s...",
+                            variantTypeString, numNegativeTrainingAndVariantType, annotationNames.size(), annotationNames));
+                    final File negativeTrainingAnnotationsFile = LabeledVariantAnnotationsData.subsetAnnotationsToTemporaryFile(
+                            annotationNames, negativeTrainingAndVariantTypeAnnotations, isNegativeTrainingAndVariantType);
+                    trainAndSerializeModel(negativeTrainingAnnotationsFile, outputPrefixTag + NEGATIVE_TAG);
+                    logger.info(String.format("%s negative model trained and serialized with output prefix \"%s\".", variantTypeString, outputPrefix + outputPrefixTag + NEGATIVE_TAG));
+
+                    if (numLabeledCalibrationAndVariantType > 0) {
+                        logger.info(String.format("Re-scoring %d %s calibration sites...", numLabeledCalibrationAndVariantType, variantTypeString));
+                        final File labeledCalibrationAnnotationsFile = LabeledVariantAnnotationsData.subsetAnnotationsToTemporaryFile(annotationNames, annotations, isLabeledCalibrationAndVariantType);
+                        final File labeledCalibrationScoresFile = positiveNegativeScore(labeledCalibrationAnnotationsFile, outputPrefixTag, CALIBRATION_SCORES_HDF5_SUFFIX);
+                        logger.info(String.format("Calibration scores written to %s.", labeledCalibrationScoresFile.getAbsolutePath()));
+                    }
+                } else {
+                    throw new UserException.BadInput(String.format("Attempted to train %s negative model, " +
+                            "but no suitable sites were found in the provided annotations.", variantTypeString));
+                }
+            }
+        } else {
+            throw new UserException.BadInput(String.format("Attempted to train %s model, " +
+                    "but no suitable training sites were found in the provided annotations.", variantTypeString));
+        }
+    }
+
+    private static int numPassingFilter(List<Boolean> isPassing) {
+        return isPassing.stream().mapToInt(x -> x ? 1 : 0).sum();
```

Comment:
Use `Stream::count`

Reply:
OK, changed to `(int) isPassing.stream().filter(x -> x).count()` (and made `isPassing` final).